docs/www/configuration.md

```diff
@@ -625,6 +625,20 @@ rpk:
   # Default: ''
   coredump_dir: "/var/lib/redpanda/coredump"
 
+  # Creates a "ballast" file that can be removed in the odd case that a redpanda node
+  # runs out of space, so that the node can continue to accept commands while a user
```

Comment:
```suggestion
  # you can delete the ballast file to allow the node to resume operations and then
```

Reply:
After you delete the ballast file, at what point is it created again to allow for the process to be repeated if necessary?

Reply:
It can be created again by running `rpk redpanda tune 'ballast_file'` on the host.